src/ensembl/src/content/app/entity-viewer/shared/helpers/tests/transcripts-sorter.test.ts

```diff
@@ -119,7 +119,10 @@ const createMANETranscript = () => {
       label: 'MANE Select',
       value: 'select',
       definition: faker.lorem.sentence()
-    }
+    },
+    gencode_basic: null,
+    tsl: null,
+    appris: null
```

Comment:
The empty metadata should be generated in the `createTranscript` function itself.

Reply:
I have tried removing this but its complaining on that line.

Reply:
I have updated the test to modify the transcript metadata instead of overwriting it. Should be fine now but do let me know if this is wrong (whether we should overwrite the whole transcript instead of modifying)